template/web/jest.config.js

```diff
@@ -7,6 +7,7 @@ const customJestConfig = {
   coverageReporters: ['clover', 'json', 'lcov', 'text'],
   moduleNameMapper: {
     'rehype-pretty-code': '<rootDir>/node_modules/rehype-pretty-code',
+    '^@coinbase/onchainkit/(.*)$': '<rootDir>/node_modules/@coinbase/onchainkit/esm/$1',
```

Comment:
Needed to mock the `@coinbase/onchainkit` package in my tests.

Reply:
why???

Reply:
😭